lib/beaker-hostgenerator/data.rb

```diff
@@ -29,6 +56,873 @@ def pe_upgrade_family
       'm' => 'master',
     }
 
+    BASE_CONFIG = {
+      'HOSTS' => {},
+      'CONFIG' => {
+        'nfs_server' => 'none',
+        'consoleport' => 443,
+      }
+    }
+
+    def base_host_config
+      {
+        'pe_dir' => pe_dir(pe_version, pe_family),
+        'pe_ver' => pe_version,
+        'pe_upgrade_dir' => pe_dir(pe_upgrade_version, pe_upgrade_family),
+        'pe_upgrade_ver' => pe_upgrade_version,
+      }
+    end
+
+    def osinfo
+      {
```

Comment:
A propos a discussion I've had with @stahnma and @haus -- this sort of thing, at some point, seems like something that should be factorable out so that it's in a common place where multiple projects can use it. Also, that common place likely just loads this as data rather than writing it as code, but that's definitely not for this PR.

Carry on.